internal/configuration/schema/session.go

```diff
@@ -8,20 +8,26 @@ type RedisSessionConfiguration struct {
 	DatabaseIndex int    `mapstructure:"database_index"`
 }
 
+type RememberMeConfiguration struct {
+	Duration     int64  `mapstructure:"duration"`
+	DurationUnit string `mapstructure:"duration_unit"`
+	Refresh      bool   `mapstructure:"refresh"`
+}
+
 // SessionConfiguration represents the configuration related to user sessions.
 type SessionConfiguration struct {
-	Name   string `mapstructure:"name"`
-	Secret string `mapstructure:"secret"`
-	// Expiration in seconds
-	Expiration int64 `mapstructure:"expiration"`
-	// Inactivity in seconds
-	Inactivity int64                      `mapstructure:"inactivity"`
-	Domain     string                     `mapstructure:"domain"`
-	Redis      *RedisSessionConfiguration `mapstructure:"redis"`
+	Name               string                     `mapstructure:"name"`
+	Secret             string                     `mapstructure:"secret"`
+	Expiration         int64                      `mapstructure:"expiration"` // Expiration in seconds
+	Inactivity         int64                      `mapstructure:"inactivity"` // Inactivity in seconds
+	RememberMeDuration string                     `mapstructure:"remember_me_duration"`
+	Domain             string                     `mapstructure:"domain"`
+	Redis              *RedisSessionConfiguration `mapstructure:"redis"`
 }
 
 // DefaultSessionConfiguration is the default session configuration
 var DefaultSessionConfiguration = SessionConfiguration{
-	Name:       "authelia_session",
-	Expiration: 3600,
+	Name:               "authelia_session",
+	Expiration:         3600,
+	RememberMeDuration: "1y",
```

Comment:
Should we change this like so?

```suggestion
	RememberMeDuration: "1m",
```

Reply:
Done as discussed